packages/types/src/element/shapeElement.ts

```diff
@@ -17,10 +17,16 @@
 /**
  * Internal dependencies
  */
-import type { Element } from './element';
+import type { Element, ElementType } from './element';
 import type { Pattern } from './pattern';
 
 export interface ShapeElement extends Element {
-  backgroundColor: Pattern;
+  backgroundColor?: Pattern;
+  isBackground?: boolean;
   isDefaultBackground?: boolean;
+  type: ElementType.Shape;
+  // TODO(#12437): Figure out why shape elements end up having these properties & fix it.
```

Comment:
Added https://github.com/GoogleForCreators/web-stories-wp/issues/12437 to track this